src/asciidoctor-extensions-api.js

```diff
@@ -234,3 +245,10 @@ if (typeof Extensions.TreeProcessor !== 'undefined') {
 TreeProcessor.$$proto.process = function (block) {
   return Opal.send(this, 'process', null, toBlock(block));
 };
+
+/**
+ * @namespace
+ * @module Extensions/Preprocessor
+ */
+// eslint-disable-next-line no-unused-vars
+var Preprocessor = Extensions.Preprocessor;
```

Comment:
True but I need it to declare the documentation `@namespace` and `@module` (otherwise I can use `@link`)